spring-boot/admin/src/main/resources/application-paco.yml

```diff
@@ -1,9 +1,80 @@
+
```

Comment:
See other comment, I would suggest to remove this file from the public repo and keep it locally as `application-private-...`